tools/windows/install-help/cal/caninstall.cpp

```diff
@@ -199,7 +199,7 @@ bool canInstall(bool isDC, bool isReadOnlyDC, bool ddUserExists, bool isServiceA
             if (isUserDomainUser)
             {
                 // if it's a domain user. We need the password if the service isn't here
-                if (!ddServiceExists && !haveUserPassword)
+                if (!ddServiceExists && !haveUserPassword && !isServiceAccount)
```

Comment:
When I look through the small lenses of github diff isServiceAccount implied gMSA but I was not sure. May be we should rename the var. Or at least add the comment?

Reply:
What happens when password is provided but it is isServiceAccount? It is not obvious in this code.

Reply:
And what about standalone Managed Service Account (sMSA). We do not support it?